scripts/veracode/veracodeUpload.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+#
+###### This is a script for packaging and uploading to Veracode Scanner for security test. Mandatory for HERE SPC review.
+# Usage for OLP EDGE TS:
+# veracodeUpload.sh /workspace/build/olp-edge-ts.zip OLP_EDGE_CIq@here.com
+#
+# Required variables:
+# VERACODE_API_PWD, VERACODE_API_USER.
+# note that below hardcoded is nubmer 443849 : TS SDK app ID in Veracode Scanner https://analysiscenter.veracode.com .
+#
+# other samples:
+# veracodeUpload.sh ./iOS.ipa OLP_EDGE_CI@here.com # any mail can be specified
+# veracodeUpload.sh ./App.apk OLP_EDGE_CI@here.com
+#
+###### Usage from official Veracode API documentation https://confluence.in.here.com/display/SPC/Veracode+API :
+# veracodeUpload.sh <filname> <email> <application-id> <label>
+# Examples:
+# veracodeUpload.sh /tmp/veracode.war michal.vossberg@here.com 443849 1.0.0-RC1
+#
+LATEST_TAG=$(git describe --tags)
```

Comment:
why do you take the latest tag. Maybe better the commit hash the script used to run the scanner on?

Reply:
From my side, when I get email with title 
"SPC - Veracode results for OLP EDGE Enablers (TypeScript) Build olp-edge-ts-v0.9.1-1-g94ca2be-168 Policy Compliance=Pass Score=99" 
is better than : 
"SPC - Veracode results for OLP EDGE Enablers (TypeScript) Build olp-edge-ts-1-g94ca2be-168 Policy Compliance=Pass Score=99" 
because second does not give us any versioning information. 

This hash (g94ca2be) is random data generated by Veracode, but not by our commit hash on Github (https://github.com/heremaps/here-olp-sdk-typescript/commit/01db17a239a4292c7dfc7ec0250775e8ca8995bf).

What is your suggestion then?

Reply:
my suggestion would be  olp-edge-ts-01db17a-1-g94ca2be-168

Reply:
updated.
Chosen your suggestion as its more clear for you and Oleksii.